### PR TITLE
Add standard

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

**Comment by juliangruber** [comment]
@rmehner question about standard: The rules depend on which Ruby version is used, eg on 2.2.0 you should add trailing commas to hashes, on 2.3.0 you shouldn't. How does one fix this with multi-version CI?

**Comment by rmehner** [comment]
@juliangruber no experience with that as I've used standardrb mostly in app settings and those have target ruby versions. My gut says "aim for the lowest supported version"

**Comment by juliangruber** [comment]
If I fix it for 2.2.0 it will fail for 2.3.0, and if I fix it for 2.3.0 it will fail for 2.2.0 🤔

**Comment by rmehner** [comment]
Well, there's always the option to set everything on fire and go back to play guitar :)

It's a bit nasty, but what could work is to use the `--config` option of standardrb and use a config file per ruby version. But you'd still need to only adhere to 2.2 standards if that's the earliest version this gem supports. Alternatively we could think of dropping everything below 2.6, as older versions are EOL anyway (https://www.ruby-lang.org/en/downloads/branches)

**Comment by juliangruber** [comment]
🎸 🔥 

I wonder, does that mean testing multiple node versions with standard isn't a supported use case then? Unexpected to me that it doesn't work well out of the box.

Another option: Only run standard on one ruby version, but run the tests on all?

**Comment by rmehner** [comment]
> I wonder, does that mean testing multiple node versions with standard isn't a supported use case then? Unexpected to me that it doesn't work well out of the box.

I don't know for node, but since the whole technology is different I'm not surprised there. Also it's ages ago since i've used standard since prettier came along.

> Another option: Only run standard on one ruby version, but run the tests on all?

That's actually a good idea. Run it on the lowest ruby version we need to support (and still think about dropping older ruby versions as no one should use them anymore)

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,9 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.1.0
-  - 2.2.0
-  - 2.3.0
   - rbx-2
```
Comment (juliangruber) [inline .travis.yml:4]: Do we need to continue testing on rubinius?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+on: push
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [2.2, 2.3, 2.4, 2.5, 2.6, 2.7, jruby, jruby-head, truffleruby, truffleruby-head]
```
Comment (juliangruber) [inline .github/workflows/ci.yml:7]: I decided to drop travis as only rubinius was tested there an not on GHA, and it was already marked as allowed to fail, so not really supported.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - if: ${{ matrix.ruby == '2.5' }}
```
Comment (juliangruber) [inline .github/workflows/ci.yml:14]: this was the oldest ruby version that I could get to execute `standard`

**Comment by rmehner** [comment]
@juliangruber https://github.com/testdouble/standard#how-do-i-specify-a-ruby-version-what-is-supported have you seen this? It specifically mentions multiple Ruby versions.

**Comment by juliangruber** [comment]
Hmm that seems not to have worked 🤔

**Pull request merged**